src/main/java/com/dev/cinema/controller/CinemaHallController.java

```diff
@@ -0,0 +1,40 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.dto.request.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.response.CinemaHallResponseDto;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.mapper.CinemaHallMapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallMapper cinemaHallMapper;
+
+    @Autowired
```

Comment:
let's use the same approach everywhere - either use this annotation over the constructor - or not